components/badge/badge.css

```diff
@@ -0,0 +1,40 @@
+.badge__demo {
+  font-size: 2rem;
+  position: relative;
+  padding: 1rem 2rem;
+  width: fit-content;
+  font-family: var(--secondary-font-family);
+}
+
+.badge__demo-btn {
+  font-family: var(--secondary-font-family);
+  font-size: 2rem;
+  background-color: var(--primary-color);
+  border-color: transparent;
+  padding: 5px 10px;
+  border-radius: 5px;
+  color: var(--white-color);
+  cursor: pointer;
+}
+
+.badge__demo-txt {
+  margin-right: 0.5rem;
+}
+
+.badge {
+  background-color: var(--secondary-color);
+  color: var(--white-color);
+  font-size: 1.6rem;
+  padding: 2.5px 8px;
+  border-radius: 50%;
+  position: absolute;
+}
+
+.badge--large {
+  font-size: 2.2rem;
+  padding: 2.5px 10px;
+}
+
+.badge--position {
+  margin: -10px;
+}
```

Comment:
Try to stick to one naming convention. Here you are using '--' and in the above classes you have used '__'
It's bit confusing

Reply:
i have used '__' for sub-elements of the block while I have used '--' for the specific attributes like sizes , positions etc.